src/test/java/seedu/address/testutil/PropertyBuilder.java

```diff
@@ -95,6 +97,14 @@ public PropertyBuilder withDeadline(LocalDate deadline) {
         return this;
     }
 
+    /**
+     * Sets the {@code Client} of the {@code Property} that we are building. TEMPORARY
```

Comment:
i think it could be permanent, but idk

Reply:
Wasn't thinking straight when writing this. Somehow thought this part will be combined with appointments in the future.